mozilla/blurts-server
# Mntor 3277

# References:

Jira: MNTOR-3277



# Description
Side Nav Bar: FAQ and How it works page to open in new tab

# Screenshot (if applicable)
<img width="218" alt="Screenshot 2024-06-10 at 2 25 11 PM" src="https://github.com/mozilla/blurts-server/assets/31937894/c492a5ac-4b85-42a6-b404-c002c1a297c4">

# How to test
In storybook, navigate to Pages => Logged in => Dashboard => US user (any variation)
Clicked links should open in new tab

# Checklist (Definition of Done)
